pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -455,20 +454,7 @@ func TestSimpleDaemonSetLaunchesPods(t *testing.T) {
 // When ScheduleDaemonSetPods is enabled, DaemonSets without node selectors should
 // launch pods on every node by NodeAffinity.
 func TestSimpleDaemonSetScheduleDaemonSetPodsLaunchesPods(t *testing.T) {
-	t.Skip("disabled for 1.10")
-
-	enabled := utilfeature.DefaultFeatureGate.Enabled(features.ScheduleDaemonSetPods)
```

Comment:
I would keep the feature-gate check in case we have to disable the feature while the feature is beta.

Reply:
Sure.